### ISSUE TITLE
Add time of day-based greetings

### ISSUE THREAD

**Issue by ShadowXVII** [Humanizr/Humanizer#1290]
It would be useful to have a helper function for building Greetings, i.e.  "Good morning", "Good afternoon", "Good evening" based on the current time.

Perhaps an extension to `DateTime` and `TimeOnly` along the lines of `.ToPeriodOfDayNotation()` might be the best way to implement this. It would return "morning" or "afternoon" or "evening" which could then be adapted into a greeting.

Or, could be `.ToGreetingNotation()` perhaps?

**Comment by logiclrd** [comment]
This seems like a fun idea but also like a potentially very deep can of worms 😛 Different cultures might have different times of day, fundamentally different greetings, and all text needs to be localized. The localization of periods of the day and greetings has overlap with but isn't strictly the same thing as the language in which it's expressed.

**Comment by XVII** [comment]
Fair enough. It seems like it was partially [implemented for a time of day perspective in Spanish](https://github.com/Humanizr/Humanizer/blob/e7a142807acd9d01fe00816b9b9ee46ca4f90b47/src/Humanizer/Localisation/TimeToClockNotation/EsTimeOnlyToClockNotationConverter.cs#L57) but can understand why it's a can of worms.

Maybe instead of a greeting, could Humanizer provide a binary `AM/PM` meridian via TimeOnly?
I.e. `TimeOnly.IsAM` or `TimeOnly.IsPM` helper. This could help power the above scenario but without the complexity.

[12-hour Clock AM/PM](https://en.wikipedia.org/wiki/12-hour_clock)

**Comment by clairernovotny** [comment]
Thanks for the suggestion. Greetings and named day periods depend on culture and application-specific time thresholds, and they are not part of Humanizer’s parity-backed locale surfaces. For the narrower AM/PM request, .NET already provides culture-aware designators through `time.ToString("tt", culture)`, while `time.Hour < 12` covers the binary check. `ToClockNotation` remains Humanizer’s supported localized time-to-text API, so we’re closing this as answered/not planned.
